src/tests/test_plot.py

```diff
@@ -55,3 +57,45 @@ def test_boxplot_stats(chinook_db):
     result = plot._boxplot_stats(con, "Invoice", "Total")
 
     assert DictOfFloats(result) == DictOfFloats(expected[0])
+
+
+def test_boxplot_stats_exception(chinook_db):
+    con = duckdb.connect(database=":memory:")
+    con.execute("INSTALL 'sqlite_scanner';")
+    con.execute("LOAD 'sqlite_scanner';")
+    con.execute(f"CALL sqlite_attach({chinook_db!r});")
+
+    res = con.execute("SELECT * FROM Invoice")
+    X = res.df().Total
+    cbook.boxplot_stats(X)
+    with pytest.raises(
+        BaseException, match="whis must be a float or list of percentiles.*"
```

Comment:
typo?

Reply:
Do you mean `whis must be a float or list of percentiles.*` is typo?

This is for passing the regex in [pytest.raises](https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest.raises), since the error message we got here will be something like

```
"whis must be a float or list of percentiles." + "\nIf you need help solving this "
    "issue, send us a message: https://ploomber.io/community"
```

We need to add star causes in the end to include the community message

Reply:
Why don't you simply look for the message directly?
I'm also not sure what is `whis` if it's not an argument it looks like a typo.

Reply:
1. Because we are more like testing the error message `whis must be a float or list of percentiles` itself instead of appended community message. Also the community message is dynamic & was relied on the `ploomber-core`. In the future the `ploomber-core` changes its community message, and we need to change the here again, which might something we are not that interested in this test case. But we can do the full-match checking here if you think it's better

2. whis seems like the term `whisker` in boxplot, https://www.khanacademy.org/math/statistics-probability/summarizing-quantitative-data/box-whisker-plots/a/box-plot-review#:~:text=A%20box%20and%20whisker%20plot%E2%80%94also%20called%20a%20box%20plot,the%20box%20at%20the%20median. We do have [methods](https://github.com/ploomber/jupysql/blob/master/src/sql/plot.py#L126) has `whis` as parameter

Reply:
Ok good so it is an argument!
Fair for 1. I think it's good to go.